receiver/githubreceiver/internal/scraper/githubscraper/helpers_test.go

```diff
@@ -262,10 +262,10 @@ func TestGetAge(t *testing.T) {
 
 	for _, tc := range testCases {
 		t.Run(tc.desc, func(t *testing.T) {
-			min := time.Now()
-			max := min.Add(tc.hrsAdd).Add(tc.minsAdd)
+			_min := time.Now()
```

Comment:
what was the purpose in making these changes?

Reply:
One of the linters raised an issue with the code as it was with `min` and `max` clobbering the built-in functions. I was mostly just trying to fix everything that the linter mentioned, but I'm fine to remove this change to reduce the surface area of the PR.

Reply:
interesting. i wonder if it was a recent update to the linter since it hadn't been caught previously. fine enough to leave in